customer-bill/src/main/java/org/fiware/tmforum/customer_bill/domain/customer_bill/AppliedCustomerBillingRate.java

```diff
@@ -0,0 +1,82 @@
+package org.fiware.tmforum.customer_bill.domain.customer_bill;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.fiware.tmforum.common.domain.EntityWithId;
+import org.fiware.tmforum.customer_bill.domain.*;
+import org.fiware.tmforum.mapping.annotations.AttributeGetter;
+import org.fiware.tmforum.mapping.annotations.AttributeSetter;
+import org.fiware.tmforum.mapping.annotations.AttributeType;
+import org.fiware.tmforum.mapping.annotations.MappingEnabled;
+
+import java.net.URL;
+import java.time.Instant;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+@MappingEnabled(entityType = AppliedCustomerBillingRate.TYPE_APPLIED_CUSTOMER_BILLING_RATE)
+public class AppliedCustomerBillingRate extends EntityWithId {
+
+    public static final String TYPE_APPLIED_CUSTOMER_BILLING_RATE = "applied_customer_billing_rate";
+
+    @Getter(onMethod = @__({@AttributeGetter(value = AttributeType.PROPERTY, targetName = "href")}))
+    @Setter(onMethod = @__({@AttributeSetter(value = AttributeType.PROPERTY, targetName = "href")}))
```

Comment:
Href should be a URI, not URL. I was searching for clarification on this, too;) https://engage.tmforum.org/communities/community-home/digestviewer/viewthread?GroupId=31&MessageKey=0cbd8aee-c684-4510-8d41-17583d3b53d7&CommunityKey=d543b8ba-9d3a-4121-85ce-5b68e6c31ce5&tab=digestviewer